ydb/docs/en/core/recipes/backup-collections.md

```diff
@@ -0,0 +1,194 @@
+# Recipes and examples
```

Comment:
Too generic header. Refactor this into a recipe folder and put each scenario into a separate file with a verbose, fully matching title for SEO purposes. Link to all scenarios from index.md of the new folder and from relevant parts of other articles above.